src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -36,7 +38,57 @@
 JOBS_MISSING_ERR = "No valid job IDs provided"
 
 CELLS_NOT_PROVIDED_ERR = "cell_id_list not provided"
-DOES_NOT_EXIST = "does_not_exist"
+
+
+class OutputStateErrMsg(Enum):
```

Comment:
Decided to turn output state errors into an `Enum` and not sure if I got carried away adding instance methods and coupling that to `jm.add_errors_to_results`

Reply:
This does seem a little like an excuse to use `Enum`s... 😁

Reply:
Yes. I think `Enum` is fine but I may have gone overboard when I found out each enum could act as an instance ...